### PR TITLE
Extend the conditional gatherer docs

### PR DESCRIPTION
**Pull request by tremes**
This extends the conditional gatherer documentation.

## Categories

- [ ] Bugfix
- [ ] Enhancement
- [ ] Backporting
- [X] Others (CI, Infrastructure, Documentation)

## Sample Archive

no new data

## Documentation

- `README.md` file and `docs/conditional-gatherer/README.md` file are updated

## Unit Tests

no test needed

## Privacy

Yes. There are no sensitive data in the newly collected information.

## Breaking Changes

No

## References

https://issues.redhat.com/browse/???
https://bugzilla.redhat.com/show_bug.cgi?id=???
https://access.redhat.com/solutions/???

### CODE REVIEW

File: pkg/gatherers/clusterconfig/recent_metrics.go [hunk 1]
```diff
@@ -25,13 +25,14 @@ const (
...
 // Gathered metrics:
-// 	 virt_platform
-//   etcd_object_counts
-//   cluster_installer
-//   vsphere_node_hw_version_total
-//   namespace CPU and memory usage
-//   console_helm_installs_total
-//   followed by at most 1000 lines of ALERTS metric
+//
+//		 virt_platform
```
Comment (rluders) [inline pkg/gatherers/clusterconfig/recent_metrics.go:29]: @tremes this one here that I was talking about.

Comment (tremes) [reply pkg/gatherers/clusterconfig/recent_metrics.go:29]: thanks. Let me fix it

**Comment by tremes** [comment]
This is not a new feature. Just a documentation update.
/label docs-approved
/label px-approved

**Pull request merged**